index.js

```diff
@@ -9,7 +9,30 @@ module.exports.quaff = quaff;
 function sniff(buffer, callback) {
     if (buffer instanceof Buffer === false) return callback(invalid('Must pass in type Buffer object.'));
 
-    var head = buffer.toString().substring(0,100);
+    var header = buffer.toString().substring(0,400);
+
+    // check for topojson/geojson
+    if (header.trim().indexOf('{') == 0) {
+        if (header.indexOf('\"tilejson\":') !== -1) return callback(null, 'tilejson');
```

Comment:
@GretaCB technically there could be whitespace between a name and the `:` token ... it's not common, but e.g. `"type" : "FeatureCollection" ...` is valid JSON and handling that could dodge a future head scratcher.